Core/Src/main.c

```diff
@@ -82,7 +96,24 @@ int main(void)
      * 7. Full regression test as much as possible. If don't know what
      * regression test is, read up on it.
      */
-
+	HAL_StatusTypeDef ret;
+	uint8_t hum_val[3]; //humidity raw value

```

Comment:
Perhaps it is easier to just have a longer variable name instead of a comment explaining what it is. 
humidity_raw_value is not much longer and it makes it clearer for future references as well.